.github/workflows/cherry-picker.yml

```diff
@@ -0,0 +1,36 @@
+name: Cherry Picker
+
+on:
+  pull_request_target:
+    types: [closed]
+
+jobs:
+  cherry_picker:
+    name: Cherry Picker
+    runs-on: ubuntu-22.04
+    if: ${{ github.event.pull_request.merged == true }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install & Configure Python
+        run: |
+          sudo apt install python3.10
```

Comment:
How stable is this? Might it make sense to fetch the result of `python --version` (or something similar) to check the Python version present on the system?

Reply:
This could probably be removed, `python3.10` is the default on GitHub runners and in theory this should be forwards-compatible. I don't know why I specifically put the version in there rather than just using `python3`.